crates/sdk/src/lib.rs

```diff
@@ -268,6 +270,23 @@ impl<E: StarkFriEngine<SC>> GenericSdk<E> {
         Ok(proof)
```

Comment:
I think we can just delete `generate_root_verifier_input`?

Reply:
ok I see it's technically different from `generate_e2e` due to the wrapping. But you would only wrap if you want to do the root proof, so I still say we delete it from SDK